src/btree/bt_debug.c

```diff
@@ -1420,7 +1454,6 @@ __debug_page_row_leaf(WT_DBG *ds, WT_PAGE *page)
     WT_ROW_FOREACH (page, rip, i) {
         WT_RET(__wt_row_leaf_key(session, page, rip, ds->key, false));
         WT_RET(__debug_item_key(ds, "K", ds->key->data, ds->key->size));
-
```

Comment:
I would suggest leaving in this blank line. It separates the key and value calls.